esp/src/src-react/components/DaliAdmin.tsx

```diff
@@ -31,69 +31,80 @@ interface DaliAdminProps {
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.GetLogicalFile} itemKey="getlogicalfile" style={pivotItemStyle(size)} >
+                    <GetLogicalFile />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.GetLogicalFilePart} itemKey="getlogicalfilepart" style={pivotItemStyle(size)} >
+                    <GetLogicalFilePart />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.GetProtectedList} itemKey="getprotectedlist" style={pivotItemStyle(size)} >
+                    <GetProtectedList />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.GetValue} itemKey="getvalue" style={pivotItemStyle(size)} >
+                    <GetValue />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.SetLogicalFileAttribute} itemKey="setlogicalfilepartattr" style={pivotItemStyle(size)} >
+                    <SetLogicalFilePartAttr />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.UnlockSDSLock} itemKey="unlockSdsLock" style={pivotItemStyle(size)} >
+                    <DaliSDSUnlock />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.SetProtected} itemKey="setprotected" style={pivotItemStyle(size)} >
+                    <SetProtected />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.SetUnprotected} itemKey="setunprotected" style={pivotItemStyle(size)} >
+                    <SetUnprotected />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.SetValue} itemKey="setvalue" style={pivotItemStyle(size)} >
+                    <SetValue />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.Add} itemKey="daliadd" style={pivotItemStyle(size)} >
+                    <DaliAdd />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.Delete} itemKey="dalidelete" style={pivotItemStyle(size)} >
+                    <DaliDelete />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.Count} itemKey="dalicount" style={pivotItemStyle(size)} >
+                    <DaliCount />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.Import} itemKey="daliimport" style={pivotItemStyle(size)} >
+                    <DaliImport />
+                </PivotItem>
+                <PivotItem headerText={nlsHPCC.Export} itemKey="daliExport" style={pivotItemStyle(size)} >
+                    <Stack tokens={{ childrenGap: 12 }} styles={{ root: { maxWidth: 400 } }}>
+                        <TextField label="Path" placeholder="/your/dfs/path" value={path} onChange={(_, v) => setPath(v || "")} />
+                        <Checkbox label="Safe" checked={safe} onChange={(_, c) => setSafe(!!c)} />
+                        <DefaultButton href={`/WsDali/Export?Path=${encodeURIComponent(path)}&Safe=${safe}`} >
```

Comment:
One other thing here. You probably do actually need to have some sort of onClick on this button that checks what the path value is. Gordon mentioned he'd checked with Jake, and if the field is left blank or set to "/" this could download a very large blob of XML. So should have some sort of confirmation dialog that double-checks that the person really meant to do that.